src/app/system/system-routing.module.ts

```diff
@@ -221,11 +222,21 @@ const routes: Routes = [
       },
       {
           path: 'roles-and-permissions',
-          component: RolesAndPermissionsComponent,
-          resolve: {
-                roles: RolesAndPermissionsResolver
-          },
           data: { title:  extract('Roles and Permissions'), breadcrumb: 'Roles and Permissions' },
+          children: [
+            {
+              path: '',
+              component: RolesAndPermissionsComponent,
+              resolve: {
+                roles: RolesAndPermissionsResolver
+            }
```

Comment:
Ensure proper indentation.

Reply:
After you commented about the Accounting Service, I went to check all the other components and many of them also has the comment as Accounting Service. I've changed the ones with this to System Service in the new push.